.pre-commit-config.yaml

```diff
@@ -1,7 +1,5 @@
 default_language_version:
-  python: python3.10
-
-files: "^databuilder/"
+  python: python3.9
```

Comment:
question: Can we remove this specification entirely?

Reply:
We need to set a python version, it can either be done per rule or with a default.  We've been moving to the default pattern everywhere of late as we've not found a need to change version per rule.